src/containers/Greeting/LoginForm.tsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { Field, Form, Formik } from 'formik';
 import { useTranslation } from 'react-i18next';
+import { Snackbar } from '@material-ui/core';
```

Comment:
bad. should be imported from components
containers module should not know anything about components implementation. hide it